package.json

```diff
@@ -164,6 +166,8 @@
     "graphql": "15.8.0",
     "graphqurl": "^1.0.1",
     "identity-obj-proxy": "^3.0.0",
+    "jest": "^28.0.0-alpha.11",
```

Comment:
Should we be using [`28.0.1`](https://www.npmjs.com/package/jest/v/28.0.1) rather than [`28.0.0-alpha.11`](https://www.npmjs.com/package/jest/v/28.0.0-alpha.11)
?

Reply:
Yes, I've updated the PR.